index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+   <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- Navbar -->
+
+    <header>
```

Comment:
**[REQUIRED CHANGES]**

- To enforce the use of HTML5 semantic tag properly, the `<header>` element represents a container for introductory content or a set of navigational links in your website. It should only wrap your `hamburger` and `welcome` as used in your design. please fix this and refer to this link to read more about the [`<header>` tag](https://www.w3schools.com/tags/tag_header.asp)